crates/typed-store/src/rocks/mod.rs

```diff
@@ -1825,14 +1864,40 @@ pub fn base_db_options() -> DBOptions {
     }
 
     // The table cache is locked for updates and this determines the number
-    // of shards, ie 2^6. Increase in case of lock contentions.
+    // of shards, ie 2^10. Increase in case of lock contentions.
     opt.set_table_cache_num_shard_bits(10);
 
+    // Allow buffering 1GiB of writes before flushing to disk.
+    opt.set_write_buffer_size(256 * 1024 * 1024);
+    opt.set_max_write_buffer_number(4);
+
+    opt.set_universal_compaction_options(&UniversalCompactOptions::default());
+    // Compaction trigger for universal compaction. Default to 8.
+    opt.set_level_zero_file_num_compaction_trigger(
+        read_size_from_env(ENV_VAR_L0_NUM_FILES_COMPACTION_TRIGGER)
+            .unwrap_or(DEFAULT_L0_NUM_FILES_COMPACTION_TRIGGER)
+            .try_into()
+            .unwrap(),
+    );
+    opt.set_num_levels(8);
```

Comment:
I'm curious - what is the noticeable impact from this?

Reply:
There should not be any observable impact with the current data size. I can remove this.